engine/flink/management/src/test/scala/pl/touk/nussknacker/engine/management/FlinkRestManagerSpec.scala

```diff
@@ -133,6 +133,15 @@ class FlinkRestManagerSpec extends FunSuite with Matchers with PatientScalaFutur
     ))
   }
 
+  test("return running status if cancelled job has last-modification date later then running job") {
+    statuses = List(JobOverview("2343", "p1", 20L, 10L, JobStatus.RUNNING), JobOverview("1111", "p1", 30L, 5L, JobStatus.CANCELED))
```

Comment:
Shouldn't we switch order in this list so that the entry with canceled could be skipped?

Reply:
I didn't notice we sort statuses by last modification earlier.